assets/css/success.rtl.css

```diff
@@ -3,13 +3,13 @@
 	align-items: center;
 	flex-wrap: wrap;
 	line-height: 1;
+	padding-top: 4px;
 }
 
 .wc-payment-gateway-method-logo-wrapper img {
 	margin-left: 0.5rem;
-	padding-top: 4px;
 }
 
-.wc-payment-gateway-method-logo-wrapper.wc-payment-bnpl-logo img {
-	max-height: 30px;
```

Comment:
Previously, there was a `padding-top: 4px` style, which was being subtracted from these 30px.
Now, the container itself has a `4px` top padding (instead of the image). So I needed to subtract `4` from the original `30` in order to keep the image the same height.

The `4px` top padding was cutting off some of the images like so (notice the top rounded corners):
<img width="173" alt="Screenshot 2024-11-07 at 1 08 03 PM" src="https://github.com/user-attachments/assets/dbf8b81b-eb51-4b9b-8a06-859e2db9738e">

Reply:
Ah, yeah, this is better. Thanks!